elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```diff
@@ -38,16 +44,85 @@
             max-width: 1140px;
         }
     }
+
     [slot="tray"] {
         width: 100%;
+        min-height: 60px;
     }
-    .pfe-navigation-item__tray--container {
-        margin: 0 auto;
+
+    [slot="trigger"] {
+        margin: 0;
+    }
+
+    // Override grid so that CTAs do not overlap.
+    .pfe-navigation--footer {
```

Comment:
@chrisdo1 can you change all the `.pfe-navigation--footer` classes to `.pfe-navigation--footer:not(:empty)` so the styles only apply when CTAs are present? Thanks!

Reply:
Fixed. Thanks.

Reply:
+1 tested